lighthouse-core/gather/gatherers/inspector-issues.js

```diff
@@ -58,9 +66,31 @@ class InspectorIssues extends Gatherer {
         const issueReqId = issueDetails.request && issueDetails.request.requestId;
         if (issueReqId &&
           networkRecords.find(req => req.requestId === issueReqId)) {
-          artifact.mixedContent.push(issue.details.mixedContentIssueDetails);
+          artifact.mixedContent.push(issueDetails);
         }
       }
+      if (issue.details.sameSiteCookieIssueDetails) {
+        const issueDetails = issue.details.sameSiteCookieIssueDetails;
+        const issueReqId = issueDetails.request && issueDetails.request.requestId;
```

Comment:
any reason to filter this for ones with known network records since we end up filtering in the audit anyways for ones with defined URLs? Seems like it would be ok for the artifact to be unopinionated and just provide all issues, but I may be missing context here.

Reply:
There's a weird thing that happens where a single issue generates duplicate requests where only one of them actually has a requestId that matches in the network records. The logic here is to dedup these only take the one with a matching request. The filtering that happens in the audit just ensures we don't have duplicate urls in the subItems.